doc/man3/PKCS12_gen_mac.pod

```diff
@@ -43,17 +59,18 @@ If I<salt> is NULL then a suitable salt will be generated and used.
 If I<iter> is 1 then an iteration count will be omitted from the PKCS#12
 structure.
 
-PKCS12_gen_mac(), PKCS12_verify_mac() and PKCS12_set_mac() make assumptions
-regarding the encoding of the given passphrase. See L<passphrase-encoding(7)>
-for more information.
+PKCS12_gen_mac(), PKCS12_verify_mac(), PKCS12_set_mac() and
+PKCS12_set_pbmac1_pbkdf2() make assumptions regarding the encoding of the
+given passphrase. See L<passphrase-encoding(7)> for more information.
 
 =head1 RETURN VALUES
 
-All functions return 1 on success and 0 if an error occurred.
+All functions of integer type return 1 on success and 0 if an error occurred.
```

Comment:
Consider: _All functions returning an integer return 1 on success and 0 if an error occurred._

Reply:
Done, thanks